src/Generator.Rewrite/RewriteOptions.cs

```diff
@@ -3,36 +3,50 @@
 namespace OpenTK.Rewrite
 {
     /// <summary>
-    /// A container class used by <see cref="CommandLine.Parser" /> to parse command line arguments.
+    /// A class containing all command line options used when rewriting an assembly.
     /// </summary>
-    public class Options
+    public class RewriteOptions
     {
         /// <summary>
         /// Gets or sets the path to the target assembly that should be rewritten.
         /// </summary>
-        [Option('a', "assembly", Required = true,
-            HelpText = "The path to the target assembly that should be rewritten.")]
+        [Option(
+            'a',
+            "assembly",
+            Required = true,
+            HelpText = "The path to the target assembly that should be rewritten."
+        )]
         public string TargetAssembly { get; set; }
 
         /// <summary>
-        /// Get or sets the path to the strong name key which should be used to sign or resign the assembly.
+        /// Gets or sets the path to the strong name key which should be used to sign or resign the assembly.
         /// </summary>
-        [Option('k', "signing-key",
-            HelpText = "The path to the strong name key which should be used to sign or resign the assembly.")]
+        [Option(
+            'k',
+            "signing-key",
+            HelpText = "The path to the strong name key which should be used to sign or resign the assembly."
+        )]
         public string StrongNameKey { get; set; }
 
         /// <summary>
         /// Gets or sets a value indicating whether calls to GL.GetError() are wrapped around each native call.
         /// </summary>
-        [Option('d', "debug", Default = false,
-            HelpText = "Enable calls to GL.GetError(), wrapped around each native call.")]
+        [Option(
```

Comment:
Check formatting. Can't remember what we decided on.

Reply:
See [this comment](https://github.com/opentk/opentk/pull/783#discussion_r203193476).

I hope this is what you meant.